akka-testkit-typed/src/main/scala/akka/testkit/typed/scaladsl/TestProbe.scala

```diff
@@ -241,6 +248,24 @@ class TestProbe[M](name: String)(implicit system: ActorSystem[_]) {
     o.asInstanceOf[C]
   }
 
+  /**
+   * Expect the given actor to be stopped or stop withing the given timeout or
+   * throw an AssertionFailure.
+   */
+  def expectStopped[U](actorRef: ActorRef[U], max: FiniteDuration): Unit = {
```

Comment:
in untyped TestKit it is called `expectTerminated`, is there a reason for using another name?

Reply:
I made it different because of the different semantics from the old testkit where you have to watch first etc. Additionally since in the APIs `ctx.stop(actor)`, `PostStop`, `Behavior.stopped` I think this is easier to find.

I don't have super strong feelings about it though, so if terminated is more correct, because it is terminated after it has stopped perhaps, I can change.

Reply:
I like the `Stopped` here, had the same impression at first, but then realised that it even works without the watch. 

Maybe call this out in the docs, since many people may be migrating from old Akka?